.github/workflows/website-checker.yml

```diff
@@ -23,7 +23,7 @@ jobs:
   website-check:
     # If there's already a `type/docs-cherrypick` label or an explicit `pr/no-docs` label, we ignore this check 
```

Comment:
```suggestion
    # If there's already a `type/docs-cherrypick` and a `pr/no-docs` label, we ignore this check 
```

Reply:
I think that's not accurate though?

There are three cases of labels:
1. no matching labels => should run workflow
2. type/docs-cherry-pick => shouldn't run workflow
3. pr/no-docs => shouldn't run workflow


The conditional will evaluate to:

1. !false && !false => true && true => true
2. !true && !false => false && true => false
3. !false && !true => true && false => false

So it's accurate to say if there's a docs-cherry-pick *or* a pr-no-docs we ignore the check

Reply:
The intention was to disable the checker in the presence of _either_ label so I think we should keep the comment as is. Otherwise, it would suggest that _both_ labels are required to disable the checker.

Reply:
👍 I'll merge it but if I'm totally wrong I'm happy to edit that comment.